flutter_local_notifications_linux/lib/src/helpers.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:dbus/dbus.dart';
+
+import 'model/enums.dart';
+import 'model/hint.dart';
+
+/// Helper method for validating notification IDs.
+/// Ensures IDs are valid 32-bit integers.
+// TODO(proninyaroslav): temp solution
```

Comment:
@MaikuB 
Also pay attention to these TODOs (there are several), is it worth leaving as is?

Reply:
I'd say it's ok to remove them

Reply:
Forgot to add my thinking is that this and https://github.com/proninyaroslav/flutter_local_notifications/blob/ddc57043dcb6d68bd94d48b9fc601753ff4f899c/flutter_local_notifications_linux/lib/src/typedefs.dart#L5 could go into the platform interface and that could be done after merging this in

Reply:
Ok, I will leave this until it's included in the platform interface.